server/startup/survey.js

```diff
@@ -0,0 +1,10 @@
+Meteor.startup(function() {
+  if (Survey.find().count() === 0) {
+    // Creating a random password
+    var pass = Random.id(20);
+
+    // Creating the great admin
+     Survey.insert({online: false});
```

Comment:
Nevermind, edited that. The thing is I sometimes clone files then edit in them instead of writing from scratch :) which produces that. Excuse moi